src/lib/mina_base/account_update.ml

```diff
@@ -983,12 +983,26 @@ module Account_precondition = struct
         | Full of Zkapp_precondition.Account.Stable.V2.t
         | Nonce of Account.Nonce.Stable.V1.t
         | Accept
-      [@@deriving sexp, equal, yojson, hash, compare]
+      [@@deriving sexp, yojson, hash, compare]
 
       let to_latest = Fn.id
+
+      let to_full = function
```

Comment:
could this be defined in the definition of `equal`, since it's not used elsewhere.

If so, you wouldn't need to define it locally

Reply:
It's also used in several other places, like here:
https://github.com/MinaProtocol/mina/blob/8d572b916a67db0edd5a8597416ee2cc10be1cc5/src/lib/mina_base/account_update.ml#L1045

Reply:
thanks, I didn't see that